### PR TITLE
PDE-4264 - Adds support for subcategories

### PR DESCRIPTION
**Pull request by standielpls**
Please watch video to see what it looks like in lieu of downloading the repo and running locally:
https://cdn.zappy.app/97dbf6b4ab5ff3f9efdf448a30cb616d.gif

# How to add new subcategories
1. Create a folder under the main collection
2. Create page content under the nested folder

No need to make any changes to the _config.yml, or changing the structure of other things, the side-nav will handle all category and subcategory rendering.

### CODE REVIEW

**Comment by standielpls** [comment]
Note to self: remove this commit: https://github.com/zapier/visual-builder/pull/504/commits/d97ce0aa171f2e516f0cffc65dac562346617f76 before merging as this is a sample pages

**Review by kreddlear — DISMISSED** [review]
This looks great to me! Thanks for digging in and figuring out how to get sub categories set up, Stan!

Let me know if you need a second approval before merging when you remove the commit with the sample pages 👍

**Review by marinahand — APPROVED** [review]

**Review by kreddlear — APPROVED** [review]

**Pull request merged**
